service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -209,7 +209,7 @@ public void addStopHook(Callable<? extends CompletionStage<?>> hook) {
         ServiceInfo serviceInfo = ServiceInfo.of(serviceName);
 
         // ServiceClientLoader
-        CircuitBreakers circuitBreakers = new CircuitBreakers(actorSystem, new CircuitBreakerConfig(configuration),
+        CircuitBreakers circuitBreakers = new CircuitBreakersImpl(actorSystem, new CircuitBreakerConfig(configuration),
```

Comment:
Ah ok, in that case sent it over. I can re-add the trait myself if we decide so. 

I'm sorry for all this roundtrips. 
Thank you very much for your contribution and patience! ;-)